troubleshooting/troubleshooting.md

```diff
@@ -20,71 +20,61 @@ The flight controller attempts to stabilize the vehicle's attitude so that it is
-    ```
-    pi@raspberrypi:~ $ lsusb
-    Bus 001 Device 007: ID 26ac:0011
-    ```
-    - Or the following device name:
-    ```
-    pi@raspberrypi:~ $ ls /dev/serial/by-id/*
-    /dev/serial/by-id/usb-3D_Robotics_PX4_FMU_v2.x_0-if00
-    ```
-3. If none of this commands has a valid output, check the Pixhawk connection with the Raspberry.
-    - You can test the Pixhawk or the usb cable with your surface computer with QGroundControl.
-    - Try to change:
-        1. The usb port used in the Raspberry.
-        2. The usb cable between the Pixhawk and the Raspberry.
-        3. The Pixhawk.
-
-4. If you get a valid output in step 2, try to connect the Pixhawk in your surface computer and flash ArduSub with [ArduSub firmware update](/software/ardusub-firmware.html#updating).
-
-Make sure the companion computer is powered with a supply that is capable of delivering at least 2A.
-
-Make sure that the QGroundControl is configured to automatically connect to UDP links. Click on the 'Q' icon in the upper left to view the Application Settings. Click on the 'General Settings' tab. In the options for 'Autoconnect to:', make sure the UDP option is checked.
-
-<img src="/images/qgc-autoconnect-settings.png" class="img-responsive img-center" style="max-height:400px;">
+#### Verify Network Configuration
 
-#### Check Your Network
-
-Verify that your [network settings](/getting-started/installation.html#network-setup) are correct, your Ethernet IP address should be 192.168.2.1 and the subnet mask should be 255.255.255.0.
+Verify that your [network settings](/getting-started/installation.html#network-setup) are correct. Verify your network configuration by entering the command `ipconfig` (Windows) or `ifconfig` (Mac/Linux) on the surface computer command line. The output should show that your Ethernet IP address is 192.168.2.1 and the subnet mask is 255.255.255.0.
 
 You should be able to ping the companion computer from the surface computer. On the surface computer's command line enter:
 
 	ping 192.168.2.2
 
-If you do not get a ping response, then something is wrong with the network communication between the surface computer and the companion computer.
+If you do not get a ping response, then something is wrong with the network communication between the surface computer and the companion computer. You may have to adjust your firewall and/or antivirus settings to allow QGrouncControl access to the network.
 
-Check your network settings. The surface computer should have a static IP address of 192.168.2.1. You may have to adjust your firewall settings to allow QGrouncControl access to the network. 
+#### Firewall and Antivirus
 
-Check the activity lights on the Raspberry Pi Ethernet Jack. The lights should be on or blinking.
+Antivirus and firewall software can block the incomming connection from the ROV. Make an exception/rule to allow *inbound and outbound* traffic on UDP ports 5600 and 14550, or turn off your antivirus and firewall software.
```

Comment:
What about waterlinked ports? Should we leave that for the waterlinked docs?

Reply:
> Should we leave that for the waterlinked docs?

Yes, this troubleshooting is targeted at a lower layer (basic autopilot connection across the network).